rpm/SOURCES/etc/init.d/sth-orig-2016-03-21

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```

Comment:
I would suggest removing this file since Github itself is a control versioning system... :) But I leave it 100% to your discretion... ;)